client/python/pyproject.toml

```diff
@@ -78,11 +78,15 @@ ignore-packages = [
     "cffi",                  # MIT License (MIT)
     "click",                 # BSD-3-Clause
     "cryptography",          # Apache-2.0 or BSD-3-Clause
+    "dulwich",               # Apache-2.0 or GPL-2.0-or-later
     "fsspec",                # BSD-3-Clause
     "jaraco.functools",      # MIT License (MIT)
     "jeepney",               # MIT License (MIT)
     "more-itertools",        # MIT License (MIT)
     "mypy_extensions",       # MIT License (MIT)
+    "poetry",                # MIT License (MIT)
+    "poetry-core",           # MIT License (MIT)
```

Comment:
pip-licenses doesn't know about the new poetry version (transitively poetry-code)